scripts/copy_files.js

```diff
@@ -0,0 +1,17 @@
+const fs = require("fs");
+const path = require("path");
+
+// Define the source and destination file paths
+const sourceFile = path.join(__dirname, '../src/external/aria');
```

Comment:
Shouldn't this be a responsability of the implementer, not the SDK?

Reply:
this way we ensure telemetry is ready anywhere is consumed, by now the goal is just to simplify the build process, we could evaluate later if its needed